### DISCUSSION TITLE
How can I make purchases for real money through the api using any store?

### DISCUSSION THREAD

**Discussion by patriarcholeksiy** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#189] [Q&A]
Is it possible to purchase using the api through different stores?

**Comment by LeleDerGrasshalmi** [comment]
wym different stores?

psn/xbox etc have their own apis.

**Reply by patriarcholeksiy** [reply]
For example, the galaxy store. Does the store itself have to process purchase requests using fortnite player data, or is it done using epic games servers?

**Reply by LeleDerGrasshalmi** [reply]
that should depend how you purchase stuff (ingame or the store of the platform)
